packages/core/data-transfer/lib/providers/local-strapi-destination-provider/__tests__/index.test.ts

```diff
@@ -0,0 +1,120 @@
+import { createLocalStrapiDestinationProvider } from '../index';
+import { getStrapiFactory, getContentTypes } from '../../test-utils'
+
+describe('Local Strapi Source Destination', () => {
+    describe('Boostrap', () => {
```

Comment:
```suggestion
    describe('Bootstrap', () => {
```
We've fixed this everywhere now 😂

Reply:
Hahahaha that's the cost of copy pasting